### DISCUSSION TITLE
Any plans to add it to TechEmPower?

### DISCUSSION THREAD

**Discussion by cirospaciari** [Neoteroi/BlackSheep#307] [Q&A] [answered]
A lot has changed after Round 21
https://www.techempower.com/benchmarks/#section=test&runid=3d6c6cc0-92ff-4861-86dc-42cef0ecc50f&test=plaintext&l=hra0hr-35r

Do you mind if I add Blacksheep with gunicorn+uvicorn+httptools and also with socketify ASGI server?

**Comment by RobertoPrevato** [comment, marked as answer]
Hi @cirospaciari 
Thank You for offering help with this. The only reason `blacksheep` is not in TechEmpower benchmarks anymore is because I didn't have time to maintain that part. I'll be happy and honored if you contributed to those benchmark this way. 😄 
And congratulations on your work on socketify! I didn't try it, yet, but I'm following it.

**Reply by cirospaciari** [reply]
Thanks a lot! Congrats on Blacksheep, is a really wonderful project, I will add the benchmarks by the end of the month.
